lib/osf-components/addon/components/node-metadata-form/styles.scss

```diff
@@ -0,0 +1,81 @@
+.container {
+    background-color: #fff;
+    display: inline-flex;
+    flex-direction: column;
+
+    button,
+    svg {
+        color: #337ab7;
+
+        &:focus,
+        &:hover {
+            background: #efefef;
+            color: #000;
+        }
+    }
+
+    div {
+        display: flex;
+        justify-content: space-between;
+        margin: 5px 0;
+    }
+
+    dd {
+        margin: 14px 0;
+    }
+
+    dl {
+        margin-bottom: 0;
+    }
+
+    dt {
+        align-items: flex-end;
+        display: flex;
+        justify-content: space-between;
+        margin: 0 0 5px;
+    }
+
+    h2 {
+        font-weight: 300;
+        font-size: 30px;
+        color: #263947;
+    }
+}
+
+.contributor-list {
+    flex-direction: column;
+
+    dt > a {
+        border: 1px solid #ddd;
```

Comment:
Should use color variable `$color-border-gray` for this.

Reply:
And here.